### PR TITLE
Local

### PR DESCRIPTION
**Pull request by Darjusch**
Please review and give feedback

### CODE REVIEW

**Comment by d-pettersson** [comment]
All good from my side, no particular feedback.

**Comment by 5h3rr1ll** [comment]
Yap! All good also from my side! 👍

File: test/product.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+let chaiHttp = require('chai-http');
+let server = require('../dist/server');
+let should = chai.should();
+
+
+chai.use(chaiHttp);
+
+describe('Product', () => {
+    beforeEach((done) => {
+        Product.default.remove({}, (err) => {
```
Comment (jwdotpark) [inline test/product.js:14]: it gave me a deprecation warning - deleteOne or deleteMany might be better option (not sure it's chai thing)

**Review by jwdotpark — COMMENTED** [review]
everything working perfectly from my machine too!

**Pull request merged**